spec/controllers/articles_controller_spec.rb

```diff
@@ -19,11 +19,16 @@
 
   describe 'GET #create' do
     let(:user) { User.create(email: 'test@is.test', password: '1234Password') }
-
+    let(:post_params) do
+      { article: {
+        title: 'Some Title', link: 'www.test.test',
+        publication: { publication_id: 1 } } }
```

Comment:
Closing hash brace must be on the line after the last hash element when opening brace is on a separate line from the first hash element.